README.md

```diff
@@ -1,96 +1,58 @@
+## Obtaining Sample Data for development purposes
+Once a user has identified a set of target publications for potential analysis, using either the [ADEPT browser](https://xdd.wisc.edu/adept#loginPanel), [xDD REST API](https://xdd.wisc.edu/api/v1), the [COSMOS system](https://github.com/UW-COSMOS/cosmos), or some combination thereof, users can request a subset of these publications for local development purposes. 
 
-## Defining an input 
-In order to deploy an application against the xDD, the _type_ and _set definition_ of input data must be defined. Fundamental questions of each are:
+Requests for development datasets can be made through the [ADEPT browser](https://xdd.wisc.edu/adept) or by contacting the xDD administrative team directly by email (https://contact@geodeepdive.org). Users will then be provided a project and user-specific URL to 200 documents randomly sampled from the identified pool of documents. This link will be provided either by email or through the ADEPT browser depending on how the sample data was requested.
 
-- _Type_ : Which of the data products that xDD can provide does the application need?
-- _Set definition_ : What documents need to be processed? 
+## Defining inputs and outputs within your application
+An xDD application should have a defined `/input/` and `/output/` directory. The application should be designed such that the expected format of the `/input/` folder is identical to the [sample data](#obtaining-sample-data-for-development-purposes) provided at the beginning of the development process. 
 
-Applications can utilize any or all of the above data products. For example, an application may utilize natural language output from Stanford's CoreNLP to extract entity relationships, then read data from COSMOS-derived tables mentioning those entities.
+Output must be well-defined and meet [Terms of Service](#determining-if-xdd-is-right-for-your-project). We strongly recommend that users with any doubts about the permissability of output format reach out to the xDD team directly (contact@geodeepdive.org). *Only* items found within the `/output/` directory upon application completion will be returned to users.
 
-Because xDD's corpus is large and multi-disciplinary, it is often important to restrict the set of documents used as input. This can be accomplished in several ways:
+## Creating and testing your docker image
+UW-Madison's Center for High Throughput Compouting provides a guide for creating a docker image [here](http://chtc.cs.wisc.edu/docker-build.shtml). Full documentation for the Dockerfile specification can be found [here](https://docs.docker.com/engine/reference/builder/). CHTC also provides a guide for testing a docker image locally [here](http://chtc.cs.wisc.edu/docker-test.shtml). 
 
-1. Specifying a list of article DOIs.
-2. Supplying a list of journals to target.
-3. Filtering the corpus by keyword, phrases, publication date ranges, etc.
```

Comment:
I think it's important to communicate that these kinds of set definitions are possible. Just sending them to ADEPT or the API endpoint doesn't inform potential users what handles exist to define a dataset.

Reply:
Added this back in under obtaining sample data section.